server/handlers/getters.go

```diff
@@ -31,30 +31,17 @@ func (ah *ActivityHandler) getMessageFromChat(chat *msteams.Chat, messageID stri
 	return msg, nil
 }
 
-func (ah *ActivityHandler) getReplyFromChannel(userID string, teamID, channelID, messageID, replyID string) (*msteams.Message, error) {
-	client, err := ah.plugin.GetClientForUser(userID)
-	if err != nil {
-		ah.plugin.GetAPI().LogError("unable to get bot client", "error", err)
-		return nil, err
-	}
-
-	var msg *msteams.Message
-	msg, err = client.GetReply(teamID, channelID, messageID, replyID)
+func (ah *ActivityHandler) getReplyFromChannel(teamID, channelID, messageID, replyID string) (*msteams.Message, error) {
+	msg, err := ah.plugin.GetClientForApp().GetReply(teamID, channelID, messageID, replyID)
```

Comment:
In the documentation, the permission `ChannelMessage.Read.All` for application level was already present. I don't know where it was being used before but it will be used now. We have the feature that even if a user is not connected, his messages should be visible in a linked channel using impersonation. That feature was not working before. It will work with this change.